src/frontend/web_application/.eslintrc

```diff
@@ -1,19 +1,10 @@
 {
   "extends": "airbnb",
   "parser": "babel-eslint",
-  "ecmaFeatures": {
-    "modules": true
-  },
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "ecmaFeatures": {
-      "experimentalObjectRestSpread": true
-    }
-  },
+  "root": true,
   "env": {
     "browser": true,
-    "es6": true,
-    "jasmine": true,
```

Comment:
it fails on jasmine dependency, 3 solutions:

* Ignore .spec.jsx? files
* explicit import of jasmine (but style in devdeps so it requires to ignore it)
* re-declare as global